src/app/frontend/replicasetdetail/updatereplicas.html

```diff
@@ -0,0 +1,33 @@
+<!--
+Copyright 2015 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<md-dialog aria-label="Create a new namespace" layout="column" layout-padding>
+  <md-content layout-padding>
+    <h4 class="md-title" >Set desired number of pods</h4>
+    <p>Replica set webapp will be updated to reflect the desired count.</p>
+    <form ng-submit="ctrl.updateReplicasCount()">
+      <md-input-container class="md-block">
+        <label>Number of pods</label>
+        <input type="number" min="1" ng-model="ctrl.replicas" placeholder="{{ctrl.curReplicasNr}}"
```

Comment:
I've displayed it as a placeholder. I thought about couple of short messages to display near that like `Current: 5`, but none is good enough for me.